src/components/authenticationPage.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+
+class WaitingPage extends Component {
+    
+async postData(url = '', data = {},token) {
+
+    const response = await fetch(url, {
+      method: 'POST', 
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorisation':token
+      },
+      redirect: 'follow', // manual, *follow, error
+      referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      body: JSON.stringify(data) // body data type must match "Content-Type" header
+      });
+    return response.json(); // parses JSON response into native JavaScript objects
+    }
+  
+    getUrlVars = (vars) =>   {
+        window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+        vars[key] = value;
+    });
+    
+    }
+    render() {
+        let vars = {};
+        this.getUrlVars(vars)
+        this.postData('http://localhost:8080/connect', {}, vars.access_token)
```

Comment:
in the readme ive have put that as when done terraform this will need to be changed as i don't know where its hosted in aws right now and when done terraform and envars will need to be added?